ntp-daemon/src/config/peer.rs

```diff
@@ -189,11 +233,31 @@ impl<'de> Deserialize<'de> for PeerConfig {
                             let raw: String = map.next_value()?;
 
                             let parsed_addr =
-                                NormalizedAddress::from_string(raw.as_str().to_string())
+                                NormalizedAddress::from_string_ntp(raw.as_str().to_string())
                                     .map_err(de::Error::custom)?;
 
                             addr = Some(parsed_addr);
                         }
+                        "ke_addr" => {
```

Comment:
Do we want this to be a different name? For the end user there isn't really a need to differentiate here and it might be a bit confusing.